docs/source/osds/scenarios.rst

```diff
@@ -171,185 +196,204 @@ An example of using the ``non-collocated`` OSD scenario with encryption, bluesto
      - /dev/sdd
      - /dev/sdd
 
+
+.. _osd_scenario_lvm:
```

Comment:
I actually think the LVM section should come before the others since they're deprecated